README.md

```diff
@@ -129,6 +129,7 @@ If you need a client that offers more customization such as advanced cache confi
   - [Testing and mocking](#testing-and-mocking)
   - [Other]
     - [Request interceptors](#request-interceptors)
+    - [abortController](#abortController)
```

Comment:
please use the right casing. this is a JS class name, and its name is `AbortController`